vendor/engines/split_accounts/app/models/split_accounts/split_account.rb

```diff
@@ -36,10 +39,30 @@ def extra_penny_count
     #   subaccounts.where("expires_at IS NOT NULL").order(expires_at: :asc).first
     #
     def earliest_expiring_subaccount
-      subaccounts = splits.map{ |split| split.subaccount }.select(&:expires_at)
+      subaccounts = splits.map(&:subaccount).select(&:expires_at)
```

Comment:
Or actually, master has `splits.map(&:subaccount).compact.min_by(&:expires_at)` since subaccount could be `nil` on part of `create`.

That wouldn't necessarily work for `suspended_at` since `suspended_at` could definitely be `nil`.